app/views/organisations/training.njk

```diff
@@ -24,6 +24,15 @@
 
       {% set headingHtml %}
         {% include "_includes/page-heading.njk" %}
+<p class="govuk-body">Tell applicants why they should train with you. You could mention:</p>
+
+          <ul class="govuk-list govuk-list--bullet">
+            <li>your key values</li>
```

Comment:
could we remove the word 'key'? Maybe we could say 'important' or just 'your values' would still make sense. I always remember learning that you should only use the word 'key' if you're referring to an actual set of keys in the GOV.UK style guide.